sys/fs/nfsserver/nfs_nfsdport.c

```diff
@@ -3695,7 +3696,7 @@ nfssvc_srvcall(struct thread *p, struct nfssvc_args *uap, struct ucred *cred)
 		if (!error)
 			error = nfsrv_adminrevoke(&adminrevoke, p);
 	} else if (uap->flag & NFSSVC_DUMPCLIENTS) {
-		error = copyin(uap->argp, (caddr_t)&dumplist,
+		error = copyincap(uap->argp, (caddr_t)&dumplist,
```

Comment:
```suggestion
		error = copyincap(uap->argp, &dumplist,
```
is surely fine? Though the cast is upstream...

Reply:
I left it in to minimize the diff to upstream, though as part of this exercise I have been tempted to do a sweep removing (caddr_t) casts upstream.

Reply:
Remove pointless casts from copyin/out seems like a good thing to do eventually, but I'm fine leaving this here for now.